tutorials/arvo/clay.md

```diff
@@ -4,6 +4,9 @@ weight = 4
 template = "doc.html"
 aliases = ["/docs/learn/arvo/clay/"]
 +++
+
+*Current docs on clay are out of date. Until they are updated this remains for primarily historical purposes.*
```

Comment:
Tweaking language/formatting

> Current docs on clay are out of date. Until they are updated, this doc remains for historical purposes.

Please generalize this to other boilerplate messages